feat(docker): add support for URL environment variable (#2879)
swagger-api/swagger-editor: README.md

This will run Swagger Editor (in detached mode) on port 80 on your machine, so you can open it by navigating to `http://localhost` in your browser.  

> * You can provide the link of a remote API document to be loaded by the editor on startup (may not be available if some security policies such as CSP or CORS are enforced):

Reviewer: ```suggestion
* You can provide a URL pointing to an API definition (may not be available if some security policies such as CSP or CORS are enforced):
```
